user-guide/Advanced_Modules/Correlation/Tutorials/Correlation_Tutorial_Element_Timeout.md

```diff
@@ -0,0 +1,102 @@
+---
+uid: Correlation_Tutorial_Element_Timeout
+---
+
+# Responding to element timeouts using Correlation
+
+In this tutorial, you will learn how to use Correlation to take action when an element is in timeout for longer than 5 minutes.
+
+Expected duration: 15 minutes
+
+> [!NOTE]
+> The content and screenshots for this tutorial were created in DataMiner 10.4.6.
+
+## Prerequisites
+
+- A DataMiner System that is [connected to dataminer.services](xref:Connecting_your_DataMiner_System_to_the_cloud).
+
+## Overview
+
+This tutorial consists of the following steps:
+
+- [Step 1: Deploy the 'Correlation KATA Switch Timeout' item from the DataMiner Catalog](#step-1-deploy-the-correlation-kata-switch-timeout-item-from-the-dataminer-catalog)
+- [Step 2: Create the Correlation rule](#step-2-create-the-correlation-rule)
+- [Step 3: Generate a timeout alarm](#step-3-generate-a-timeout-alarm)
+- [Step 4: Verify whether the rule actions have been executed](#step-4-verify-whether-the-rule-actions-have-been-executed)
+
+## Step 1: Deploy the 'Correlation KATA Switch Timeout' item from the DataMiner Catalog
+
+1. Look up the 'Correlation KATA Switch Timeout' item in the DataMiner Catalog.
+
+1. Deploy the catalog item to your DataMiner Agent by clicking the *Deploy* button.
```

Comment:
Maybe say what element is installed and where, so they know where to find it back